presidio-analyzer/tests/conf/recognizers.yaml

```diff
@@ -0,0 +1,24 @@
+recognizers:
```

Comment:
NIT: instead of having this copied several times, maybe have links to this file from the readme and code?

Reply:
This file is used for testing, not sure we want to reference to it (if tests change and docs don't for example)